package.json

```diff
@@ -596,6 +640,11 @@
                     "default": false,
                     "description": "Print executed code in REPL.",
                     "scope": "window"
+                },
+                "julia.documentation.darkMode": {
+                    "type": "boolean",
+                    "default": true,
+                    "markdownDescription": "Use [Documenter.jl](https://github.com/JuliaDocs/Documenter.jl)'s dark theme stylesheet for documentation pane, otherwise its light theme stylesheet will be used."
```

Comment:
afaict no, look at [this comment](https://github.com/julia-vscode/julia-vscode/pull/1308#pullrequestreview-440156230):

> light/dark mode needs to be switched manually; this is because Documenter.jl's stylesheet is supposed to be applied to html itself rather than body, so we can't use this vscode webview style API

But I feel @pfitzseb may have an idea to solve this.